src/rendertron.ts

```diff
@@ -22,15 +22,34 @@ export class Rendertron {
   private port = process.env.PORT || this.config.port;
   private host = process.env.HOST || this.config.host;
 
+  async createRenderer(
+    config: Config,
+    onRendererCreated: (renderer: Renderer) => void
+  ) {
+    const browser = await puppeteer.launch({
+      headless: false,
+      args: ['--no-sandbox']
+    });
+
+    browser.on('disconnected', () => {
+      this.createRenderer(config, onRendererCreated);
+    });
+
+    const renderer = new Renderer(browser, config);
```

Comment:
Should we just substitute the argument for the callback with this instead of this extra line. What do you think?

Reply:
I think it is a bit simpler, yes!